Makefile

```diff
@@ -50,21 +50,19 @@ bootstrap: check-docker
 build-binary:
 	GOOS=linux GOARCH=amd64 go build -ldflags ${LDFLAGS} -o $(BINARY_DEST_DIR)/logger main.go
 
-build: build-with-container docker-build
+build: bootstrap build-with-container docker-build
```

Comment:
I'm not crazy about `bootstrap` implicitly being part of the `build` target. `bootstrap` can conceivably take a long time (even when no updates are actually being made). Contributors can code / build / test faster if `bootstrap` isn't always creating drag.